src/components/fashiontv/LiveFeedGrid.jsx

```
@@ -0,0 +1,501 @@
+      return `https://www.youtube.com/embed/live_stream?channel=${channelId}`;
+    }
+    return stream.embedUrl;
+  };
+
+  const renderTwitchEmbed = (stream) => {
+    const channelName = stream.url.match(/twitch\.tv\/([^/?]+)/)?.[1];
+    return (
+      <div className="w-full h-full">
+        <div id={`twitch-embed-${stream.id}`} className="w-full h-full"></div>
+        <script 
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)